packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -508,7 +508,7 @@ module.exports = function(log, config, oauthdb) {
     const templateName = 'verifyShortCode';
     const metricsTemplateName = 'verify';
     const code = message.code;
-    const subject = gettext('Verification code: %(code)s');
+    const subject = gettext('Account authorization code');
```

Comment:
After user testing they wanted to keep this, I think we can sentence casing the original?

Objections @ryanfeeley ?

Reply:
Ah, right-o, I was just going from the Google Doc.

Reply:
This should be back how you want it now, @vbudhram.